source/beamline/SGM/SGMBeamline.cpp

```diff
@@ -202,6 +207,10 @@ void SGMBeamline::setupBeamlineComponents()
 	hexapod_ = new SGMHexapod(this);
 	// Rotate the hexapod system to the plane of the sample plate.
 	hexapod_->rotateSystem(45, 0, 105);
+	// Control set for scan configurations.  Probably need a better solution.
+	hexapodControlSet_ = new AMControlSet(this);
+	hexapodControlSet_->addControl(hexapod_->xAxisPrimeControl());
+	hexapodControlSet_->addControl(hexapod_->zAxisPrimeControl());
```

Comment:
@iainworkman @dretrex 
For the hexapod, is the scanning plane X'-Z' or is it X'-Y'?